python/hail/methods/impex.py

```diff
@@ -764,13 +764,179 @@ def import_gen(path, sample_file=None, tolerance=0.2, min_partitions=None, chrom
            reference_genome=nullable(GenomeReference))
 def import_table(paths, key=[], min_partitions=None, impute=False, no_header=False,
                  comment=None, delimiter="\t", missing="NA", types={}, quote=None, reference_genome=None):
+    """Import delimited text file (text table) as key table.
+
+    The resulting key table will have no key columns, use :meth:`.KeyTable.key_by`
+    to specify keys.
+
+    Examples
+    --------
+
+    Given this file
+
+    .. code-block:: text
+
+        $ cat data/samples1.tsv
+        Sample     Height  Status  Age
+        PT-1234    154.1   ADHD    24
+        PT-1236    160.9   Control 19
+        PT-1238    NA      ADHD    89
+        PT-1239    170.3   Control 55
+
+    The interesting thing about this table is that column ``Height`` is a
+    floating-point number, and column ``Age`` is an integer. Hail can impute
+    these types or we can state the explicitly:
+
+    Importing a table with explicit type annotations:
+
+    >>> table = hc1.import_table('data/samples1.tsv', types={'Height': TFloat64(), 'Age': TInt32()})
+
+    Note that string columns like ``Sample`` and ``Status`` do not need to be
+    typed, because ``String`` is the default type.
+
+    Imputing a table with type imputation (the file must be read twice to
+    impute):
+
+    >>> table = hc1.import_table('data/samples1.tsv', impute=True)
+
+    **Detailed examples**
```

Comment:
Do we allow such headers now?

Reply:
I just loaded the first file fine in this branch. What do you think the problem is?

Reply:
Hail context is gone. Should be `methods.import_table`